core/spec.md

```diff
@@ -379,10 +379,19 @@ existing entity can be deleted.
     start with an alphanumeric character and be unique within the scope of
     this map
   - Values MAY be empty strings
-  - When serialized as a string (e.g. in an HTTP header), the serialization
-    SHOULD be in JSON without line-breaks
+  - When serialized as an HTTP header, each "name" MUST appear as a separate
+    HTTP header prefixed with `xRegistry-labels-` and the header value
+	MUST be the label's "value".
```

Comment:
Any reason for this four-space indent?

Reply:
Nope - a tab slipped in there. I'll remove them.